src/lodepng/lodepng.cpp

```diff
@@ -2063,9 +2047,9 @@ The parameters are of type short, but should come from unsigned chars, the short
 are only needed to make the paeth calculation correct.
 */
 static unsigned char paethPredictor(short a, short b, short c) {
-  short pa = abs(b - c);
-  short pb = abs(a - c);
-  short pc = abs(a + b - c - c);
+  short pa = LODEPNG_ABS(b - c);
```

Comment:
AFAIK abs is sometimes faster than doing a direct comparison depending on how smart the compiler is, so I will leave it as-is.